packages/xrpl/src/models/transactions/NFTokenMint.ts

```diff
@@ -86,8 +86,11 @@ export interface NFTokenMint extends BaseTransaction {
    *
    * This field must be hex-encoded. You can use `convertStringToHex` to
    * convert this field to the proper encoding.
+   *
+   * This field must not be an empty string. Omit it from the transaction or
+   * set to `undefined` value if you do not use it.
    */
-  URI?: string
+  URI?: string | null
```

Comment:
I don't think the type needs to be changed here - according to [this StackOverflow answer](https://stackoverflow.com/questions/65912702/can-an-optional-property-value-be-null), `?` is equivalent to adding ` | undefined` to a type.

Reply:
This is done because of explicit null in the unit test as we have added it as something that dev can pass over as a valid value.